src/gui/ar/groundPlaneAnchors.js

```diff
@@ -21,13 +28,20 @@ createNameSpace("realityEditor.gui.ar.groundPlaneAnchors");
     let selectedMeshName = null;
     let constrainToX = false;
     let constrainToZ = false;
+    let initialAnchorPosition = null;
+    let initialLocalMatrix = null;
 
     let originColor = 0xffffff;
     let xBoxColor = 0xff0000;
     let zBoxColor = 0x0000ff;
     let selectionColor = 0xffff00;
-    let mouseCursorColor = 0xff00ff;
+    let mouseCursorColor = 0xffffff;
     let mouseCursorMesh = null;
+    let initialCalculationMesh = null;
+
+    const REALTIME_DRAG_UPDATE = true; // TODO: this looks a bit jittery for now, turn on if we improve its performance
```

Comment:
Is this meant to be on or off? The comment seems to imply off.

Reply:
Oh the comment is out of date, will fix – thanks!